v2-src/start_inspector_mac.spec

```diff
@@ -6,7 +6,7 @@ block_cipher = None
 a = Analysis(['start_inspector.py'],
              pathex=['/Users/dhuang/gdrive/dev/iot-inspector-client/v2-src'],
```

Comment:
I don't know if this line has any use at all, especially given that you can compile without any errors...

Reply:
Agreed!